torch/csrc/jit/script/init.cpp

```diff
@@ -476,6 +476,35 @@ void initJitScriptBindings(PyObject* module) {
             }
             return result;
           })
+      .def(
+          "save",
+          [](std::shared_ptr<Function> self,
+             const std::string& filename,
+             const ExtraFilesMap& _extra_files = ExtraFilesMap()) {
+            Module module;
+            // Make a graph with a fake self argument
+            auto graph = self->graph()->copy();
+            graph->insertInput(0, "self");
```

Comment:
The type of input 0 is not set here (it is also never checked, which is why it doesn't fail).